### PR TITLE
Introduce basic level terminologies

### PR DESCRIPTION
**Pull request by mohamed7genius**
* Add more terminologies
* Change terminologies structure to contain the level (level represents the minimal level required to have a reasonable expectation that such a person is familiar with the word and its implications.) and `en` translation for now.

### CODE REVIEW

File: contents/hanafi/en/4_ruling_bleeding_most_likely_hayd.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 - Your latest recorded discharges may soon qualify as Hayd, although they haven't yet surpassed the 72-hour threshold.
 - Everytime the bleeding discontinues such that you are not sure if it will come back later or if it has completely stopped, it's Wajib to complete your Salaah at the latest permissible times:
-  - For Fajr and Dhuhur: Pray at the last possible moment.
+  - For Fajr and Duhr: Pray at the last possible moment.
```
Comment (Ajwah) [inline contents/hanafi/en/4_ruling_bleeding_most_likely_hayd.md:7]: Dhuhr

Dh stands for ظ

Comment (mohamed7genius) [reply contents/hanafi/en/4_ruling_bleeding_most_likely_hayd.md:7]: Isn't it better to use `Zuhr` as Z for ظ 
I'm not sure which is easier for users as there're a lot of spellings

> Zuhr prayer: It is also transliterated Dhuhr, Duhr, Thuhr or Luhar.
From https://en.wikipedia.org/wiki/Zuhr_prayer

File: src/components/Content/terminologies.json [hunk 2]
```diff
@@ -1,16 +1,22 @@
 {
   "ahkaam": "rulings",
+  "asr": "afternoon prayer",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:3]:
```suggestion
  "asr": {
     "clarification": "afternoon prayer",
     "level": "basic"
   }
```

File: src/components/Content/terminologies.json [hunk 3]
```diff
@@ -1,16 +1,22 @@
 {
   "ahkaam": "rulings",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:2]: expert

File: src/components/Content/terminologies.json [hunk 4]
```diff
@@ -1,16 +1,22 @@
...
+  "asr": "afternoon prayer",
+  "duhr": "midday prayer",
+  "fajr": "dawn prayer", 
   "fard": "obligatory",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:6]: advanced

File: src/components/Content/terminologies.json [hunk 5]
```diff
@@ -1,16 +1,22 @@
...
   "ghusl": "full ablution",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:7]: basic

File: src/components/Content/terminologies.json [hunk 6]
```diff
@@ -1,16 +1,22 @@
...
   "hajj": "pilgrimage",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:8]: basic

File: src/components/Content/terminologies.json [hunk 7]
```diff
@@ -1,16 +1,22 @@
...
   "hayd": "menstruation",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:9]: advanced

File: src/components/Content/terminologies.json [hunk 8]
```diff
@@ -1,16 +1,22 @@
...
+  "isha": "night prayer",
   "istihadah": "dysfunctional bleeding",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:11]: advanced

File: src/components/Content/terminologies.json [hunk 9]
```diff
@@ -1,16 +1,22 @@
...
   "kabah": "the sacred structure located in the center of the Masjid al-Haram in Mecca, Saudi Arabia",
   "ma’dhur": "excused person",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:13]: expert

File: src/components/Content/terminologies.json [hunk 10]
```diff
@@ -1,16 +1,22 @@
...
+  "maghrib": "sunset prayer",
   "makrooh": "disliked",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:15]: advanced

File: src/components/Content/terminologies.json [hunk 11]
```diff
@@ -1,16 +1,22 @@
...
   "mustahabb": "recommended",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:16]: advanced

File: src/components/Content/terminologies.json [hunk 12]
```diff
@@ -1,16 +1,22 @@
...
   "nifaas": "postnatal bleeding",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:17]: advanced

File: src/components/Content/terminologies.json [hunk 13]
```diff
@@ -1,16 +1,22 @@
...
   "qadaa": "makeup",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:18]: advanced

File: src/components/Content/terminologies.json [hunk 14]
```diff
@@ -1,16 +1,22 @@
...
+  "ramadan": "fasting month",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:19]: basic

File: src/components/Content/terminologies.json [hunk 15]
```diff
@@ -1,16 +1,22 @@
...
   "salaah": "prayer",
   "shari’iyyah": "Islamic law",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:21]: expert

File: src/components/Content/terminologies.json [hunk 16]
```diff
@@ -1,16 +1,22 @@
...
   "sunnah": "the teachings, practices, and examples set forth by the Prophet SAW",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:22]: basic

File: src/components/Content/terminologies.json (hunk 2 above)
Comment (Ajwah) [reply src/components/Content/terminologies.json:3]: Do it like this.
`level` represents the minimal level required to have a reasonable expectation that such a person is familiar with the word and its implications. In total there are 4 levels:

* newcomer
* basic
* advanced
* expert

So the word `'Asr` only needs to be clarified for a `newcomer` whereas from basic onward it does not need clarification

File: src/components/Content/termsHelper.js
Comment (Ajwah) [inline src/components/Content/termsHelper.js:1]: Generally, you do not want to maintain separate lists as that makes it easy to forget.

File: contents/hanafi/en/4_ruling_bleeding_most_likely_hayd.md (hunk 1 above)
Comment (Ajwah) [reply contents/hanafi/en/4_ruling_bleeding_most_likely_hayd.md:7]: Yeah, transliterations are a complicated thing unfortunately. There is a cultural component to it. French speaking vs English speaking vs Malaysian:

Dhuhr
Dzuhur
Zuhr

Most likely we will need to introduce cultural transliteration renderings also:
```json
  "dhuhr": {
	     "clarification": "midday prayer",
	     "level": "basic",
              "transliterations": {
                 "ar": "الظهر",
                 
                 "en": "dhuhr",
                  "ur": "zuhr",
                  ....
	   },
```

The above are colloquial transliterations. You also have the academic format: 
* https://www.loc.gov/catdir/cpso/romanization/arabic.pdf
* https://en.wikipedia.org/wiki/Bikdash_Arabic_Transliteration_Rules

So there will be a lot more work in the future to get all of this incorporated. This is just a first step

File: src/components/Content/terminologies.json [hunk 17]
```diff
@@ -1,23 +1,110 @@
...
-  "shari’iyyah": "Islamic law",
-  "sunnah": "the teachings, practices, and examples set forth by the Prophet SAW",
-  "takbeer-e-tahreema": "the opening phrase of the prayer",
-  "tawaaf": "circumambulation around the Kabah",
-  "tuhr": "purity",
-  "umrah": "minor pilgrimage",
-  "wajib": "necessary",
-  "wudu": "partial ablution"
+  "ahkaam": {
+    "clarification": "rulings",
```
Comment (Ajwah) [inline src/components/Content/terminologies.json:3]: let's future proof translations:
```suggestion
    "clarification": {
       "en": "rulings",
     }
```

**Review by Ajwah — APPROVED** [review]

**Pull request merged**